pkg/events/group.go

```diff
@@ -0,0 +1,200 @@
+package events
+
+import (
+	"sync"
+
+	"github.com/aquasecurity/tracee/pkg/errfmt"
+	"github.com/aquasecurity/tracee/pkg/logger"
+)
+
+// TODO: add states to the EventGroup struct (to keep states of events from that group)
+type EventState struct {
+	Submit uint64 // event that should be submitted to userspace (by policies bitmap)
```

Comment:
why make this public with get/sets?

Reply:
This will be in the next parts of the refactor (this PR got too big). You can consider this PR a "1st phase". Its hard to make all changes in a single PR (it would take quite sometime and its better to have the refactor rolling so other PRs can rebase and suggest things/changes. 

This EventState will be a full encapsulated type, etc etc.